spike_main/spike_device.cc

```diff
@@ -35,7 +35,7 @@ struct vAddr_info{
 
 
 static void help(int exit_code = 1){
-  fprintf(stderr, "now you are in help function");
```

Comment:
Is it necessary to shut the fprintf?

Reply:
> Is it necessary to shut the fprintf?
I think we'd better disable other printf log ohther than fprintf for file logging, that is right?